branch_previous/download.py

```diff
@@ -0,0 +1,33 @@
+from exercise_utils.file import create_or_update_file, append_to_file
+from exercise_utils.git import add, commit
+
+__resources__ = {"expected-tree.png": "expected-tree.png"}
```

Comment:
This does not need to be here if we preserve the original markdown Mermaid code block!